libretro/libretro.cpp

```diff
@@ -1272,7 +1275,12 @@ bool retro_load_game(const struct retro_game_info *game)
    Core_SetGraphicsContext(ctx);
    SetGPUBackend((GPUBackend)g_Config.iGPUBackend);
 
+#ifdef IOS
+   // also use emu thread for vulkan
+   useEmuThread              = ctx->GetGPUCore() != GPUCORE_SOFTWARE;
+#else
```

Comment:
All this really changes is which thread is the "primary" and which is "secondary".  Either way, GL comands and execution of the CPU happen on separate threads.  It's better to keep it the same as Vulkan on other platforms to avoid crashes that are specific to Vulkan/libretro/iOS.

-[Unknown]